overlord/ifacestate/ifacestate_test.go

```diff
@@ -3719,6 +3719,7 @@ slots:
 }
 
 func (s *interfaceManagerSuite) TestCheckInterfacesDenySkippedIfNoDecl(c *C) {
+	deviceCtx := s.TrivialDeviceContext(c, nil)
```

Comment:
this test probably needs to be renamed for clarity

Reply:
Done.

Reply:
should this be ...AllowBasedOnSnapTypeNoSnapDecl  to correspond more to the new test?  otherwise s/DenySkipped/NoDeny/ ?